steam-redirector/main.c

```diff
@@ -124,7 +119,7 @@ int MAIN(argc, argv) {
 	}
 
 	fprintf(stdout, "Launching '"PATHSUBST"'\n", exe_path);
-	execute(exe_path, arg);
+	execute(exe_path);
```

Comment:
Referencing our previous conversation here to make things a bit easier: https://github.com/Furglitch/modorganizer2-linux-installer/issues/25#issuecomment-3032063025